modules/installation-special-config-raid.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+[id="installation-special-config-raid_{context}"]
+== Configuring a RAID-enabled data volume
+
+You can enable software RAID partitioning to provide an external data volume. {product-title} supports RAID 0, RAID 1, RAID 4, RAID 5, RAID 6, and RAID 10 for software backup, data protection, and fault tolerance. See "About disk mirroring" for more details.
```

Comment:
Let's not say `software backup` here; RAID isn't appropriate for backups.  `rm -rf` on a RAID 1 will happily remove both copies.